packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -113,6 +113,13 @@ export type AgentLogin = {
   loginOption: LoginOption;
 };
 
+export interface StateChange {
+  state: string;
+  auxCodeId: string;
+  lastStateChangeReason?: string;
+  agentId?: string;
```

Comment:
agent id we can pass from config

Reply:
Need a bit more info here

Reply:
fallback to `this.agentConfig.agentId}` default
        ```data: {...data, agentId: data.agentId || this.agentConfig.agentId},```